StellarDotnetSdk/LedgerKeys/LedgerKey.cs

```diff
@@ -22,6 +22,16 @@ public static LedgerKey ClaimableBalance(byte[] balanceId)
         return new LedgerKeyClaimableBalance(balanceId);
     }
 
+    /// <summary>Constructs a new <c>LedgerKeyClaimableBalance</c> from the given hex-encoded claimable balance ID.</summary>
+    /// <param name="balanceId">
+    ///     Hex-encoded ID of the claimable balance entry.
+    ///     For example: <c>d1d73327fc560cc09f54a11c7a64180611e1f480f3bf60117e41d19d9593b780</c>.
+    /// </param>
+    public static LedgerKey ClaimableBalance(string balanceId)
```

Comment:
balanceIdHexString or balanceIdHash? what is the term used elsewhere in the SDK? If the correct term is found please also update the exception description and test variable names.

Reply:
`hexString` is used. So, I updated it to `balanceIdHexString`.